sdk/core/azure-core/tests/test_streaming.py

```diff
@@ -178,6 +179,18 @@ def test_decompress_compressed_header(http_request):
     decoded = content.decode('utf-8')
     assert decoded == "test"
 
+@pytest.mark.parametrize("http_request", HTTP_REQUESTS)
+def test_decompress_compressed_header_offline(port, http_request):
```

Comment:
Do we want to test aiohttp as well?

Reply:
Yes. We will add async test in another PR.